omod/src/main/java/org/openmrs/module/locationbasedaccess/fragment/controller/field/LocationsFragmentController.java

```diff
@@ -1,16 +1,37 @@
 package org.openmrs.module.locationbasedaccess.fragment.controller.field;
 
+import org.apache.commons.lang3.StringUtils;
 import org.openmrs.Location;
+import org.openmrs.Patient;
+import org.openmrs.PersonAttribute;
+import org.openmrs.PersonAttributeType;
+import org.openmrs.module.locationbasedaccess.PatientSearchAdviser;
 import org.openmrs.ui.framework.fragment.FragmentModel;
 import org.openmrs.api.context.Context;
+import org.springframework.web.bind.annotation.RequestParam;
+
 import javax.servlet.http.HttpSession;
 import java.util.List;
 
 public class LocationsFragmentController {
 
     public void controller(FragmentModel model,
-                           HttpSession session) {
+                           HttpSession session,
+                           @RequestParam(value = "patientId", required = false) Patient patient) {
         List<Location> activeLocations = Context.getLocationService().getAllLocations();
         model.addAttribute("activeLocations", activeLocations);
+
+        model.addAttribute("selectedLocationUuid", null);
+        if (patient instanceof Patient) {
+            String locationAttributeUuid = Context.getAdministrationService().getGlobalProperty(PatientSearchAdviser.LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME);
```

Comment:
I think now is the time to put LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME in the LocationBasedAccessConstants

Reply:
Sure !!!